Podman commands in chapters 2 to 6 (#85)
hsf-training/hsf-training-docker: _episodes/04-file-io.md

pwd
ls
cat io_example.txt

> echo "This was written inside the Container" >> io_example.txt

Reviewer: the container (lowercase)

~~~
{: .source}

>## Permission issues
>If you run into a `Permission denied` error, there is a simple and quick fix to continue with the exercise:
>~~~bash
>exit  # exit container

> >chmod a+w io_example.txt  # add write permissions for all users

Reviewer 2: below, "docker cp" -> "podman cp"

~~~
This was written on local host

> This was written inside the Container

Reviewer: container (if changed above)

>Note that SELinux is enabled if the output of the command `getenforce status` is `Enforcing`.
>To fix the permission issue, append `:z` (lowercase!) at the end of the mount option, like this:
>```bash
>podman run --rm -it -v $PWD:/home/docker/data:z ...
>```

> >If this still does not fix the issue you can disable SELinux by running `sudo setenforce 0`, or you can try using `sudo` to execute docker/podman commands, but both of these methods are not recommended.

Reviewer 2: ```suggestion
>If this still does not fix the issue you can disable SELinux by running `sudo setenforce 0`, or you can try using `sudo` to execute docker/podman commands, but neither of these methods is recommended.
```

Reviewer 2: It would be better to add an explanation about _why_ they are not recommended, but this is a bit beyond the scope of this PR.
